Modules/FV0/src/DigitQcTask.cxx

```diff
@@ -278,6 +277,18 @@ void DigitQcTask::initialize(o2::framework::InitContext& /*ctx*/)
   mHistCycleDurationNTF = std::make_unique<TH1D>("CycleDurationNTF", "Cycle Duration;;time [TimeFrames]", 1, 0, 2);
   mHistCycleDurationRange = std::make_unique<TH1D>("CycleDurationRange", "Cycle Duration (total cycle range);;time [ns]", 1, 0, 2);
 
+  if (auto timeGate = mCustomParameters.find(string("gateTimeForRatioHistogram")); timeGate != mCustomParameters.end()) {
+    mTimeGate = stof(timeGate->second);
+  }
+  char gateTimeRatioTitle[50] = "Ratio of events between time -";
+  char gateTimeRatioTitle2[20] = " and ";
+  char timeGateChar[8];
+  sprintf(timeGateChar, "%d", mTimeGate);
+  strcat(gateTimeRatioTitle, timeGateChar);
+  strcat(gateTimeRatioTitle2, timeGateChar);
+  strcat(gateTimeRatioTitle, gateTimeRatioTitle2);
```

Comment:
I suggest to use `std::string` + `std::to_string` or ROOT function `Form` instead of C-style string operations. It will take 1 line of the code.

Reply:
ok. Agree with Artur.

Reply:
Agree. Done in the latest commit.